woocommerce/woocommerce
# Fix #49982 - "Creation of dynamic property WC_Beta_Tester::$wporg_data is deprecated" on PHP 8.2

Fixes #49982

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Declare WC_Beta_Tester property $wporg_data to avoid deprecation notices on PHP 8.3 and above

Closes #49982 .



### How to test the changes in this Pull Request:

1. Activate the WooCommerce Beta Tester on a WordPress install running PHP 8.2 or above
2. Go to WP Admin Dashboard - Updates
3. The deprecation notice does not show up anymore



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message

Fix "Creation of dynamic property WC_Beta_Tester::$wporg_data is deprecated" on PHP 8.2

</details>
